packages/ibm-products/src/components/Coachmark/styles/_coachmark-overlay.scss

```diff
@@ -0,0 +1,327 @@
+          top: $vertical-push;
+          right: $right;
+          transform: rotate(90deg);
+        }
+      }
+
+      &-bottom {
+        transform: translate(
+          $translate-x,
+          calc(-1 * (100% - $distance-offset))
+        );
+        .#{$block-class}__caret {
+          right: $right;
+          bottom: $vertical-push;
+          transform: rotate(90deg);
+        }
+      }
+    }
+
+    &--right {
+      $left: calc(-1 * $caret-center * 1.5);
+      $translate-x: calc($distance-offset + $caret-center);
+      $vertical-push: calc($distance-offset - ($caret-height * 0.5));
+
+      transform: translate($translate-x, -50%);
+      .#{$block-class}__caret {
+        top: calc(50% - ($caret-height * 0.5));
+        left: $left;
+        transform: rotate(-90deg);
+      }
+
+      &-top {
+        transform: translate($translate-x, calc(-1 * $distance-offset));
+        .#{$block-class}__caret {
+          top: $vertical-push;
+          left: $left;
+          transform: rotate(-90deg);
+        }
+      }
+
+      &-bottom {
+        transform: translate(
+          $translate-x,
+          calc(-1 * (100% - $distance-offset))
+        );
+        .#{$block-class}__caret {
+          bottom: $vertical-push;
+          left: $left;
+          transform: rotate(-90deg);
+        }
+      }
+    }
+
+    // ANIMATED MEDIA
+    &__element-stepped-media {
+      height: calc($spacing-07 + $spacing-12); // 32 + 96 = 128
+      margin-bottom: $spacing-05;
+    }
+
+    // BODY
+    &__body {
+      padding: $spacing-01 $spacing-05 $spacing-05;
+
+      .bx--btn--ghost {
+        color: $inverse-link;
+
+        &:hover {
+          color: $inverse-link;
+        }
+      }
+
+      // CAROUSEL ELEMENTS
+      #{$block-class}-element {
+        display: block;
+        max-width: 100%;
+        flex: 0 0 100%;
+
+        #{$block-class}-element__content {
+          padding-bottom: $spacing-05;
+        }
+
+        #{$block-class}-element__title > div {
+          margin: 0 0 $spacing-03;
+
+          > h2 {
+            @include carbon--type-style('productive-heading-02');
+
+            order: 1;
+          }
+        }
+      }
+    }
+
+    // FOOTER
+    &__footer {
+      display: flex;
+      align-items: center;
+      justify-content: flex-end;
+    }
+    /* ------------------ */
+
+    // THEME
+    &__light {
+      .#{$block-class}__body {
+        .bx--link {
+          color: $inverse-link;
+
+          &:hover {
+            color: #a6c8ff;
+          }
+
+          &:active {
+            color: $text-04;
+          }
+
+          &:focus {
+            outline-color: $text-04;
+          }
+        }
+
+        .bx--btn--ghost {
+          &:hover {
+            background-color: $inverse-hover-ui;
+          }
+
+          &:active {
+            background-color: $active-secondary;
+          }
+        }
+      }
+    }
+
+    &__dark {
+      .#{$block-class}__body {
+        .bx--link {
```

Comment:
I see a couple instances of `bx`. Can you use the the `$carbon-prefix` variable in these places instead. You can import it from `ibm-products/src/global/styles/project-settings`.

Reply:
ahhh, good call! Will do